ur_moveit_config/doc/index.rst

```diff
@@ -23,3 +23,16 @@ interaction using
 
 Now you should be able to use the MoveIt Plugin in rviz2 to plan and execute trajectories with the
 robot as explained `here <https://moveit.picknik.ai/main/doc/tutorials/quickstart_in_rviz/quickstart_in_rviz_tutorial.html>`_.
+
+.. note::
+   Usually, MoveIt uses trajectory execution monitoring (TEM). If executing a trajectory takes too long,
+   e.g. because the action server died or the robot is blocked, this will lead to an error in
+   trajectory execution. However, this isn't necessarily compatible with the scaled joint
+   trajectory controller (as long as TEM is not aware of the scaling), as execution time can
+   be unbounded on purpose. Hence, TEM is disabled in the example MoveIt configuration:
```

Comment:
Another suggestion:

```suggestion
The MoveIt configuration provided here has Trajectory Execution Monitoring *disabled*, as the Scaled Joint Trajectory Controller may cause trajectories to be executed at a lower velocity than they were originally planned by MoveIt. MoveIt's TEM however is not aware of this deliberate slow-down due to scaling and will in most cases unnecessarily (and unexpectedly) abort goals.

Until this incompatibility is resolved, the default value for `execution_duration_monitoring` is set to `false`. Users who wish to temporarily (re)enable TEM at runtime (for use with other, non-scaling controllers) can do so using the ROS 2 parameter services supported by MoveIt.
```

a bit more text, but perhaps more specific about what's going on, what would happen if it wasn't disabled, which parameter is specifically responsible for disabling it and how users could (temporarily perhaps) re-enable it at runtime.

Reply:
That sounds good, thank you!